i18n/en.json

```diff
@@ -2998,6 +2998,46 @@
     "id": "app.cluster.404.app_error",
     "translation": "Cluster API endpoint not found."
   },
+  {
```

Comment:
Let's also remove the keys from the store layer which got removed. They are not required anymore.